proto/groups/v1alpha/groups.proto

```diff
@@ -0,0 +1,204 @@
+syntax = "proto3";
+
+package com.mintter.groups.v1alpha;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "mintter/backend/genproto/groups/v1alpha;groups";
+
+// Groups service exposes the functionality for managing group entities.
+// This API uses fully-qualified EID as group IDs in all the requests.
+service Groups {
+  // Creates a new group.
+  rpc CreateGroup(CreateGroupRequest) returns (Group);
+
+  // Gets a group by ID.
+  rpc GetGroup(GetGroupRequest) returns (Group);
+
+  // Updates a group.
+  rpc UpdateGroup(UpdateGroupRequest) returns (Group);
+
+  // Lists members of a group.
+  rpc ListMembers(ListMembersRequest) returns (ListMembersResponse);
```

Comment:
This call should not be necessary if we had the members inside the `Group` Message. Why do you think it's better this way? To keep the memberlist private? Don't you thing group and listmembers can get out of sync?

Reply:
It's separate because groups can potentially have hundreds of members, and we would want to pagínate them. There's no easy/clear way to paginate members if they would exist inside of the Group type.

The same reasoning is used for content.